fortio_main.go

```diff
@@ -113,12 +123,19 @@ func fortioLoad() {
 		usage("Error: fortio load needs a url or destination")
 	}
 	url := flag.Arg(0)
+	duration := *durationFlag
 	prevGoMaxProcs := runtime.GOMAXPROCS(*goMaxProcsFlag)
-	fmt.Printf("Fortio %s running at %g queries per second, %d->%d procs, for %v: %s\n",
-		fhttp.Version, *qpsFlag, prevGoMaxProcs, runtime.GOMAXPROCS(0), *durationFlag, url)
+	fmt.Printf("Fortio %s running at %g queries per second, %d->%d procs",
+		fhttp.Version, *qpsFlag, prevGoMaxProcs, runtime.GOMAXPROCS(0))
+	if *endlessFlag {
+		fmt.Printf(", until interrupted: %s\n", url)
+		duration = 0 // Endless mode is determined by having duration value equal to 0
```

Comment:
maybe we can just use that ? like -qps 0 means infinite/max qps; -t 0

yes, let's do that - it's simpler no ?
(remove the new flag altogether, just add to the help text 0 for infinite - you can still have the if for the nice message like happens for qps 0 too)

Reply:
I initially thought about the 0 value for infinite but then figured it's the same value as the default so tried a different flag.

Your suggestion below for a negative t for infinite is good alternative.